### PR TITLE
149 file names in download links

### LINKED ISSUE #149: File names in download links
## Description
As a user, I want to see the file names of the additional documents that I want to download so that I know what type of files I am downloading to my computer. 

## Acceptance criteria
- [x] In the tab "övriga filer" all the additional files are shown by their name and filetype.

### PR DESCRIPTION
**Pull request by martinjonsson01**
## Summary
Makes the download links for the additional documents show the name of the original document file.

chrome_A5no2J0W14

## Definition of done
- [x] All acceptance criteria are completed.
- [x] The code has (at least) 80% branch test coverage.
- [x] All tests pass.
- [ ] The code is reviewed and accepted by (at least) two other team members.
- [x] Views adhere to design style.

Closes #149

### CODE REVIEW

File: src/Client/Pages/Admin/ContractForm.razor [hunk 1]
```diff
@@ -94,11 +112,11 @@
 
     private Contract _contract = CreateEmptyContract();
 
-    private MultipartFormDataContent _supplierLogoContent = new MultipartFormDataContent();
+    private MultipartFormDataContent _supplierLogoContent = new();
```
Comment (Gigset) [inline src/Client/Pages/Admin/ContractForm.razor:115]: What is the purpose of removing the type here?

File: src/Client/Pages/Contracts/ContractDetails.razor [hunk 2]
```diff
@@ -42,15 +42,17 @@
...
                             <Title>
                                 <h3>Övriga filer</h3>
                             </Title>
                             <Body>
-                            <a id="additional-document" href="@Contract.AdditionalDocument" download>
-                                Ladda ned fil
+                            <a id="additional-document"
+                               href="@Contract.AdditionalDocument.Path"
+                               download="@Contract.AdditionalDocument.Name">
+                                Ladda ned @Contract.AdditionalDocument.Name
```
Comment (Gigset) [inline src/Client/Pages/Contracts/ContractDetails.razor:55]: Wow very cool!

**Review by Gigset — APPROVED** [review]
Good job, lgtm!

File: src/Client/Pages/Admin/ContractForm.razor (hunk 1 above)
Comment (martinjonsson01) [reply src/Client/Pages/Admin/ContractForm.razor:115]: It's redundant to specify the type twice!

**Review by Matte02 — APPROVED** [review]
🤩

**Review by jonathan-lindqvist — APPROVED** [review]
LGTM

**Pull request merged**